ibis/backends/polars/tests/test_select.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from ibis.backends.polars.tests.conftest import BackendTest
+
+pytest.importorskip("polars")
+
+
+def test_select(alltypes, alltypes_df):
+    df = alltypes.execute()
+    BackendTest.assert_frame_equal(df, alltypes_df)
+
+
+def test_where_multiple_conditions(alltypes, alltypes_df):
+    expr = alltypes.filter(
+        [
+            alltypes.float_col > 0,
+            alltypes.smallint_col == 9,
+            alltypes.int_col < alltypes.float_col * 2,
+        ]
+    )
+    result = expr.execute()
+
+    expected = alltypes_df[
+        (alltypes_df['float_col'] > 0)
+        & (alltypes_df['smallint_col'] == 9)
+        & (alltypes_df['int_col'] < alltypes_df['float_col'] * 2)
+    ]
+
+    BackendTest.assert_frame_equal(result, expected)
```

Comment:
Do we need these tests? This functionality should be covered in the backend suite so we don't have to duplicate tests.

Reply:
Removing

Reply:
Removed.